components/ciphertool/src/main/java/org/wso2/ciphertool/utils/Utils.java

```diff
@@ -440,12 +452,19 @@ public static String getDeploymentFilePath() {
     /**
      * Get default.json file path.
      *
-     * @return default JSON file path
+     * @return Default JSON file path.
      */
     public static Path getDefaultJSONFilePath() {
+
         String homeFolder = System.getProperty(Constants.CARBON_HOME);
-        return Paths.get(homeFolder, Constants.REPOSITORY_DIR,
-                Constants.RESOURCES_DIR, Constants.CONF_DIR, Constants.DEFAULT_JSON_FILE);
+        Path filePath = null;
+        try {
+            filePath = Paths.get(homeFolder, Constants.REPOSITORY_DIR,
+                    Constants.RESOURCES_DIR, Constants.CONF_DIR, Constants.DEFAULT_JSON_FILE);
+        } catch (InvalidPathException e) {
+            System.err.println("Error while resolving the default.json file path" + e.toString());

```

Comment:
Shall we use a logger to print error logs? Using System.err is not a good practice.

Reply:
Look for other places as well

Reply:
Since it is an interactive tool, most of the lines should be written back to the command prompt for the user to interact with.